definitions/mlops-machine_learning_model/definition.yml

```diff
@@ -17,12 +17,29 @@ synthesis:
           entityTagName: providerExternalId
         aws.sagemaker.endpoints.modelmetrics.MonitoringSchedule:
           entityTagName: monitoringSchedules
+    - identifier: modelName
+      name: modelName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: metricType
+          value: model_metric
+    - identifier: modelName
+      name: modelName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: metricType
+          value: data_metric
+    - identifier: modelName
+      name: modelName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: InferenceData
 dashboardTemplates:
-  newRelic:
+  AWS:
     template: dashboard.json
 goldenTags:
-  - providerAccountId
-  - providerAccountName
+  - instrumentation.provider
```

Comment:
wondering why the accountId and name are not golden anymore.

Reply:
this is because they are not provided for all types of rules

Reply:
that's fine. the entities that don't have it will show nothing and the ones that have it will show them.

Infra-host does a similar thing, they have like 20 golden tags but only 4-5 show depending on where the data comes from.